model/src/test/kotlin/AdvisorRecordTest.kt

```diff
@@ -66,11 +74,142 @@ class AdvisorRecordTest : StringSpec({
             has_issues: false
         """.trimIndent()
 
-        val record = shouldNotThrowAny { yamlMapper.readValue<AdvisorRecord>(yaml) }
-        record.advisorResults should haveSize(1)
-        record.advisorResults.entries.single().let { (id, results) ->
-            id shouldBe Identifier("type:namespace:name:version")
-            results shouldHaveSize 1
+            val record = shouldNotThrowAny { yamlMapper.readValue<AdvisorRecord>(yaml) }
+            with(record.advisorResults) {
```

Comment:
Nit: It seems a bit odd to use `with` here but not above. But actually, IMO using `with` does not pay off here really, and something like `this should haveSize(1)` reads a bit off to me.

Reply:
There seems to be no agreement between the reviewers here. I just want to note that this piece of code has not actually been changed by this PR. It appears in the diff because I have changed the test class from extending `StringSpec` to `WordSpec`, and therefore indentation changed. So should I revert to the original code?